spec/factories/addresses.rb

```diff
@@ -3,7 +3,7 @@
     kind { 'home' }
     sequence(:address_1, 1111) { |n| "#{n} Awesome Street NE" }
     sequence(:address_2, 111) { |n| "##{n}" }
-    city { 'Washington' }
+    city { 'Washington dc' }
```

Comment:
you no longer need this.

Reply:
Yeah, you are right, I missed adding that file.